### PR TITLE
Service initiation page: new next/back buttons

### PR DESCRIPTION
**Pull request by zachfalsetto**
issue #970

### CODE REVIEW

**Comment by nschonni** [comment]
I'm guessing the absolute paths to `http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/` should actually be replaced with relative paths.

**Comment by zachfalsetto** [comment]
Yes that will be fixed.

**Comment by thomasgohard** [comment]
The markup for the previous/next links should follow the "aligned links" pattern from the style guide: http://wet-boew.github.io/wet-boew-styleguide/v4/design/pagination-en.html

File: site/pages/servicecomplexe/index-en.hbs [hunk 1]
```diff
@@ -43,11 +43,11 @@
 		<h2>[Sub heading]</h2>
 		<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsa hic minima magni nesciunt nam. Facilis perferendis, adipisci voluptates, enim illo perspiciatis molestias itaque iste qui distinctio asperiores quidem voluptatum sunt.</p>
 		</section>
-		<nav role="navigation" class="row pagntn-prv-nxt">
+		<nav role="navigation" class="row">
 			<h2 class="wb-inv">Document navigation</h2>
 			<ul class="list-unstyled">
 				{{#next}}
-				<li class="col-sm-6 col-sm-push-6 text-right"><a rel="next" href="{{link}}"><span class="glyphicon glyphicon-chevron-right"></span><span class="pgntn-lbl">Next</span> {{title}}</a></li>
+				<li><div class="col-xs-12"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page2.html" class="btn btn-primary"><span></span>Next<span class="wb-inv"> proceed to the next page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/index-en.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/index-fr.hbs [hunk 2]
```diff
@@ -43,11 +43,11 @@
 		<h2>[Sous-titre]</h2>
 		<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsa hic minima magni nesciunt nam. Facilis perferendis, adipisci voluptates, enim illo perspiciatis molestias itaque iste qui distinctio asperiores quidem voluptatum sunt.</p>
 		</section>
-		<nav role="navigation" class="row pagntn-prv-nxt">
+		<nav role="navigation" class="row">
 			<h2 class="wb-inv">Navigation dans le document</h2>
 			<ul class="list-unstyled">
 				{{#next}}
-				<li class="col-sm-6 col-sm-push-6 text-right"><a rel="next" href="{{link}}"><span class="glyphicon glyphicon-chevron-right"></span><span class="pgntn-lbl">Suivant</span> {{title}}</a></li>
+				<li><div class="col-xs-12"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page2-fr.html" class="btn btn-primary"><span></span>Suivant<span class="wb-inv"> proceed to the next page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/index-fr.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/page2-en.hbs [hunk 3]
```diff
@@ -46,14 +46,14 @@
 		<h2>[Sub heading]</h2>
 		<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsa hic minima magni nesciunt nam. Facilis perferendis, adipisci voluptates, enim illo perspiciatis molestias itaque iste qui distinctio asperiores quidem voluptatum sunt.</p>
 		</section>
-		<nav role="navigation" class="row pagntn-prv-nxt">
+		<nav role="navigation" class="row">
 			<h2 class="wb-inv">Document navigation</h2>
 			<ul class="list-unstyled">
 				{{#previous}}
-				<li class="col-sm-6"><a rel="prev" href="{{link}}"><span class="glyphicon glyphicon-chevron-left"></span><span class="pgntn-lbl">Previous </span> {{title}}</a></li>
+				<li><div class="col-xs-9"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/index-en.html" class="btn btn-default"><span></span>Previous<span class="wb-inv"> return to the previous page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/page2-en.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/page2-en.hbs [hunk 4]
```diff
@@ -46,14 +46,14 @@
...
 				{{/previous}}
 				{{#next}}
-				<li class="col-sm-6 text-right"><a rel="next" href="{{link}}"><span class="glyphicon glyphicon-chevron-right"></span><span class="pgntn-lbl">Next</span> {{title}}</a></li>
+				<li><div class="col-xs-3"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page3.html" class="btn btn-primary"><span></span>Next<span class="wb-inv"> proceed to the next page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/page2-en.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/page2-fr.hbs [hunk 5]
```diff
@@ -46,14 +46,14 @@
 		<h2>[Sous-titre]</h2>
 		<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsa hic minima magni nesciunt nam. Facilis perferendis, adipisci voluptates, enim illo perspiciatis molestias itaque iste qui distinctio asperiores quidem voluptatum sunt.</p>
 		</section>
-		<nav role="navigation" class="row pagntn-prv-nxt">
+		<nav role="navigation" class="row">
 			<h2 class="wb-inv">Navigation dans le document</h2>
 			<ul class="list-unstyled">
 				{{#previous}}
-				<li class="col-sm-6"><a rel="prev" href="{{link}}"><span class="glyphicon glyphicon-chevron-left"></span><span class="pgntn-lbl">Précedent</span> {{title}}</a></li>
+				<li><div class="col-xs-9"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/index-fr.html" class="btn btn-default"><span></span>Précedent<span class="wb-inv"> return to the previous page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/page2-fr.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/page2-fr.hbs [hunk 6]
```diff
@@ -46,14 +46,14 @@
...
 				{{/previous}}
 				{{#next}}
-				<li class="col-sm-6 text-right"><a rel="next" href="{{link}}"><span class="glyphicon glyphicon-chevron-right"></span><span class="pgntn-lbl">Suivant</span> {{title}}</a></li>
+				<li><div class="col-xs-3"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page3-fr.html" class="btn btn-primary"><span></span>Suivant<span class="wb-inv"> proceed to the next page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/page2-fr.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/page3-en.hbs [hunk 7]
```diff
@@ -46,14 +46,14 @@
 		<h2>[Sub heading]</h2>
 		<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsa hic minima magni nesciunt nam. Facilis perferendis, adipisci voluptates, enim illo perspiciatis molestias itaque iste qui distinctio asperiores quidem voluptatum sunt.</p>
 		</section>
-		<nav role="navigation" class="row pagntn-prv-nxt">
+		<nav role="navigation" class="row">
 			<h2 class="wb-inv">Document navigation</h2>
 			<ul class="list-unstyled">
 				{{#previous}}
-				<li class="col-sm-6"><a rel="prev" href="{{link}}"><span class="glyphicon glyphicon-chevron-left"></span><span class="pgntn-lbl">Previous </span> {{title}}</a></li>
+				<li><div class="col-xs-9"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page2-en.html" class="btn btn-default"><span></span>Previous<span class="wb-inv"> return to the previous page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/page3-en.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/page3-en.hbs [hunk 8]
```diff
@@ -46,14 +46,14 @@
...
 				{{/previous}}
 				{{#next}}
-				<li class="col-sm-6 text-right"><a rel="next" href="{{link}}"><span class="glyphicon glyphicon-chevron-right"></span><span class="pgntn-lbl">Next</span> {{title}}</a></li>
+				<li><div class="col-xs-3"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page4-en.html" class="btn btn-primary"><span></span>Next<span class="wb-inv"> proceed to the next page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/page3-en.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/page3-fr.hbs [hunk 9]
```diff
@@ -46,14 +46,14 @@
 		<h2>[Sous-titre]</h2>
 		<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsa hic minima magni nesciunt nam. Facilis perferendis, adipisci voluptates, enim illo perspiciatis molestias itaque iste qui distinctio asperiores quidem voluptatum sunt.</p>
 		</section>
-		<nav role="navigation" class="row pagntn-prv-nxt">
+		<nav role="navigation" class="row">
 			<h2 class="wb-inv">Navigation dans le document</h2>
 			<ul class="list-unstyled">
 				{{#previous}}
-				<li class="col-sm-6"><a rel="prev" href="{{link}}"><span class="glyphicon glyphicon-chevron-left"></span><span class="pgntn-lbl">Précedent</span> {{title}}</a></li>
+				<li><div class="col-xs-9"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page2-fr.html" class="btn btn-default"><span></span>Précedent<span class="wb-inv"> return to the previous page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/page3-fr.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/page3-fr.hbs [hunk 10]
```diff
@@ -46,14 +46,14 @@
...
 				{{/previous}}
 				{{#next}}
-				<li class="col-sm-6 text-right"><a rel="next" href="{{link}}"><span class="glyphicon glyphicon-chevron-right"></span><span class="pgntn-lbl">Suivant</span> {{title}}</a></li>
+				<li><div class="col-xs-3"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page4-fr.html" class="btn btn-primary"><span></span>Suivant<span class="wb-inv"> proceed to the next page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/page3-fr.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/page4-en.hbs [hunk 11]
```diff
@@ -46,14 +46,14 @@
 		<h2>[Sub heading]</h2>
 		<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsa hic minima magni nesciunt nam. Facilis perferendis, adipisci voluptates, enim illo perspiciatis molestias itaque iste qui distinctio asperiores quidem voluptatum sunt.</p>
 		</section>
-		<nav role="navigation" class="row pagntn-prv-nxt">
+		<nav role="navigation" class="row">
 			<h2 class="wb-inv">Document navigation</h2>
 			<ul class="list-unstyled">
 				{{#previous}}
-				<li class="col-sm-6"><a rel="prev" href="{{link}}"><span class="glyphicon glyphicon-chevron-left"></span><span class="pgntn-lbl">Previous </span> {{title}}</a></li>
+				<li><div class="col-xs-9"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page3-en.html" class="btn btn-default"><span></span>Previous<span class="wb-inv"> return to the previous page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/page4-en.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/page4-en.hbs [hunk 12]
```diff
@@ -46,14 +46,14 @@
...
 				{{/previous}}
 				{{#next}}
-				<li class="col-sm-6 text-right"><a rel="next" href="{{link}}"><span class="glyphicon glyphicon-chevron-right"></span><span class="pgntn-lbl">Next</span> {{title}}</a></li>
+				<li><div class="col-xs-3"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page5.html" class="btn btn-primary"><span></span>Next<span class="wb-inv"> proceed to the next page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/page4-en.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/page4-fr.hbs [hunk 13]
```diff
@@ -46,14 +46,14 @@
 		<h2>[Sous-titre]</h2>
 		<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsa hic minima magni nesciunt nam. Facilis perferendis, adipisci voluptates, enim illo perspiciatis molestias itaque iste qui distinctio asperiores quidem voluptatum sunt.</p>
 		</section>
-		<nav role="navigation" class="row pagntn-prv-nxt">
+		<nav role="navigation" class="row">
 			<h2 class="wb-inv">Navigation dans le document</h2>
 			<ul class="list-unstyled">
 				{{#previous}}
-				<li class="col-sm-6"><a rel="prev" href="{{link}}"><span class="glyphicon glyphicon-chevron-left"></span><span class="pgntn-lbl">Précedent</span> {{title}}</a></li>
+				<li><div class="col-xs-9"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page3-fr.html" class="btn btn-default"><span></span>Précedent<span class="wb-inv"> return to the previous page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/page4-fr.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/page4-fr.hbs [hunk 14]
```diff
@@ -46,14 +46,14 @@
...
 				{{/previous}}
 				{{#next}}
-				<li class="col-sm-6 text-right"><a rel="next" href="{{link}}"><span class="glyphicon glyphicon-chevron-right"></span><span class="pgntn-lbl">Suivant</span> {{title}}</a></li>
+				<li><div class="col-xs-3"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page5-fr.html" class="btn btn-primary"><span></span>Suivant<span class="wb-inv"> proceed to the next page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/page4-fr.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/page5-en.hbs [hunk 15]
```diff
@@ -46,14 +46,14 @@
 		<h2>[Sub heading]</h2>
 		<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsa hic minima magni nesciunt nam. Facilis perferendis, adipisci voluptates, enim illo perspiciatis molestias itaque iste qui distinctio asperiores quidem voluptatum sunt.</p>
 		</section>
-		<nav role="navigation" class="row pagntn-prv-nxt">
+		<nav role="navigation" class="row">
 			<h2 class="wb-inv">Document navigation</h2>
 			<ul class="list-unstyled">
 				{{#previous}}
-				<li class="col-sm-6"><a rel="prev" href="{{link}}"><span class="glyphicon glyphicon-chevron-left"></span><span class="pgntn-lbl">Previous </span> {{title}}</a></li>
+				<li><div class="col-xs-9"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page4-en.html" class="btn btn-default"><span></span>Previous<span class="wb-inv"> return to the previous page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/page5-en.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/page5-en.hbs [hunk 16]
```diff
@@ -46,14 +46,14 @@
...
 				{{/previous}}
 				{{#next}}
-				<li class="col-sm-6 text-right"><a rel="next" href="{{link}}"><span class="glyphicon glyphicon-chevron-right"></span><span class="pgntn-lbl">Next</span> {{title}}</a></li>
+				<li><div class="col-xs-3"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page6.html" class="btn btn-primary"><span></span>Next<span class="wb-inv"> proceed to the next page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/page5-en.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/page5-fr.hbs [hunk 17]
```diff
@@ -46,14 +46,14 @@
 		<h2>[Sous-titre]</h2>
 		<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsa hic minima magni nesciunt nam. Facilis perferendis, adipisci voluptates, enim illo perspiciatis molestias itaque iste qui distinctio asperiores quidem voluptatum sunt.</p>
 		</section>
-		<nav role="navigation" class="row pagntn-prv-nxt">
+		<nav role="navigation" class="row">
 			<h2 class="wb-inv">Navigation dans le document</h2>
 			<ul class="list-unstyled">
 				{{#previous}}
-				<li class="col-sm-6"><a rel="prev" href="{{link}}"><span class="glyphicon glyphicon-chevron-left"></span><span class="pgntn-lbl">Précedent</span> {{title}}</a></li>
+				<li><div class="col-xs-9"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page4-fr.html" class="btn btn-default"><span></span>Précedent<span class="wb-inv"> return to the previous page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/page5-fr.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/page5-fr.hbs [hunk 18]
```diff
@@ -46,14 +46,14 @@
...
 				{{/previous}}
 				{{#next}}
-				<li class="col-sm-6 text-right"><a rel="next" href="{{link}}"><span class="glyphicon glyphicon-chevron-right"></span><span class="pgntn-lbl">Suivant</span> {{title}}</a></li>
+				<li><div class="col-xs-3"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page6-fr.html" class="btn btn-primary"><span></span>Suivant<span class="wb-inv"> proceed to the next page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/page5-fr.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/page6-en.hbs [hunk 19]
```diff
@@ -43,11 +43,11 @@
 		<h2>[Sub heading]</h2>
 		<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsa hic minima magni nesciunt nam. Facilis perferendis, adipisci voluptates, enim illo perspiciatis molestias itaque iste qui distinctio asperiores quidem voluptatum sunt.</p>
 		</section>
-		<nav role="navigation" class="row pagntn-prv-nxt">
+		<nav role="navigation" class="row">
 			<h2 class="wb-inv">Document navigation</h2>
 			<ul class="list-unstyled">
 				{{#previous}}
-				<li class="col-sm-6"><a rel="prev" href="{{link}}"><span class="glyphicon glyphicon-chevron-left"></span><span class="pgntn-lbl">Previous </span> {{title}}</a></li>
+				<li><div class="col-xs-9"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page5-en.html" class="btn btn-default"><span></span>Previous<span class="wb-inv"> return to the previous page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/page6-en.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

File: site/pages/servicecomplexe/page6-fr.hbs [hunk 20]
```diff
@@ -43,11 +43,11 @@
 		<h2>[Sous-titre]</h2>
 		<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsa hic minima magni nesciunt nam. Facilis perferendis, adipisci voluptates, enim illo perspiciatis molestias itaque iste qui distinctio asperiores quidem voluptatum sunt.</p>
 		</section>
-		<nav role="navigation" class="row pagntn-prv-nxt">
+		<nav role="navigation" class="row">
 			<h2 class="wb-inv">Navigation dans le document</h2>
 			<ul class="list-unstyled">
 				{{#previous}}
-				<li class="col-sm-6"><a rel="prev" href="{{link}}"><span class="glyphicon glyphicon-chevron-left"></span><span class="pgntn-lbl">Précedent</span> {{title}}</a></li>
+				<li><div class="col-xs-9"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page5-fr.html" class="btn btn-default"><span></span>Précedent<span class="wb-inv"> return to the previous page</span></a></div></li>
```
Comment (thomasgohard) [inline site/pages/servicecomplexe/page6-fr.hbs]: No need for absolute links as the relative link value is in the `{{link}}` variable.

**Comment by zachfalsetto** [comment]
fixed the links and the markup

**Pull request merged**